src/relayServerUtils.ts

```diff
@@ -332,6 +332,30 @@ async function convertGasToTokenAndNative(
   };
 }
 
+function secondsToDate(dateInSeconds: number) {
+  return new Date(dateInSeconds * 1000);
+}
+
+async function validateExpirationTime(
+  validUntilTime: PromiseOrValue<BigNumberish>,
```

Comment:
If the time comes un milliseconds, it will not be rejected.

Reply:
I think we agreed on not doing anything related to this. If the time is wrong, the user will see the error.